tests/modules/metamaps/classify/main.nf

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { UNTAR                } from '../../../../modules/untar/main.nf'
+include { METAMAPS_MAPDIRECTLY } from '../../../../modules/metamaps/mapdirectly/main.nf'
+include { METAMAPS_CLASSIFY    } from '../../../../modules/metamaps/classify/main.nf'
+
+workflow test_metamaps_classify {
+
+    classification_res = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data['sarscov2']['illumina']['test_paired_end_bam'], checkIfExists: true)
+    ]
+    database_folder = [
+        file(params.test_data['sarscov2']['illumina']['test_paired_end_bam'], checkIfExists: true)
+    ]
```

Comment:
should this be database ? @henningonsbring

Reply:
correct, however it is unrelated to why the tests for mapdirectly crashed

I will fix the test for classify now since I got the mapdirectly test to work